scifight/urls.py

```diff
@@ -0,0 +1,24 @@
+from django.conf.urls import url
+from scifight import views
+
+
+urlpatterns = [
+    url(r'^$',                                   views.tournament,   name='tournament'),
+    url(r'^schedule/$',                          views.schedule,     name='schedule'),
+    url(r'^fight/(?P<fight_id>[0-9]+)/$',        views.fight,        name='fight'),
+    url(r'^rooms/$',                             views.rooms,        name='rooms'),
+    url(r'^room/(?P<room_id>[0-9]+)/$',          views.room,         name='room'),
+    url(r'^teams/$',                             views.teams,        name='teams'),
+    url(r'^team/(?P<team_id_or_slug>[\w_-]+)/$', views.team,         name='team'),
+    url(r'^participants/$',                      views.participants, name='participants'),
+    url(r'^participant/(?P<part_id>[0-9]+)/$',   views.participant,  name='participant'),
+    url(r'^leaders/$',                           views.leaders,      name='leaders'),
+    url(r'^leader/(?P<leader_id>[0-9]+)/$',      views.leader,       name='leader'),
+    url(r'^juries/$',                            views.juries,       name='juries'),
+    url(r'^jury/(?P<jury_id>[0-9]+)/$',          views.jury,         name='jury'),
+    url(r'^problems/$',                          views.problems,     name='problems'),
+    url(r'^problem/(?P<problem_id>[0-9]+)/$',    views.problem,      name='problem'),
```

Comment:
But where is `(?P<tournament_slug>…)`? It's logically tied to `scifight` application, so it should be in this file, not in `scifight_proj`.